Tiltfile

```diff
@@ -41,7 +41,7 @@ providers = [
 ]
 
 for provider in providers:
-    command = '''sed -i '' -e 's@image: .*@image: '"{}"'@' ./{}/config/default/manager_image_patch.yaml'''.format(provider['image'], provider['name'])
+    command = '''sed -i'' -e 's@image: .*@image: '"{}"'@' ./{}/config/default/manager_image_patch.yaml'''.format(provider['image'], provider['name'])
```

Comment:
You can make this change on your local fork, my work station is os x though so I want the space! (this is a case of bsd sed  vs gnu sed-_-)